### ISSUE TITLE
Enum34

### ISSUE THREAD

**Issue by antnieszka** [DataMedSci/beprof#44]
On Python versions <3.4 tests fail because of this:

```
==================================== ERRORS ====================================

__________________ ERROR collecting beprof/test/test_curve.py __________________

beprof/test/test_curve.py:3: in <module>
    from beprof.curve import Curve
beprof/curve.py:1: in <module>
    from enum import IntEnum
E   ImportError: No module named enum
```

We should add [Enum34](https://pypi.python.org/pypi/enum34) backport for Pythin versions <3.4 or replace it with something else.

**Comment by grzanka** [comment]
Any idea if it can be replaced with something else ?
Its better to stick with the "batteries included" than to use external libraries.

**Comment by antnieszka** [comment]
I'm not the one who added it to this project. Suppose @matkrak didn't know about plans for Python <3.4 or 3.5.
According to this, there is not much to choose from :/ http://stackoverflow.com/questions/36932/how-can-i-represent-an-enum-in-python

**Comment by antnieszka** [comment]
As far as I can see - this enum is used only once, to represent Axis. Maybe just replace it with simple list?

**Comment by grzanka** [comment]
This enum comes probably from one of my old codes. It was quite convenient representation of axis type:  (X,Y,Z,V). I was used in general purpose methods as an argument of of function (like "calculate_average(Axis.X)". It increases readabilty and when using IntEnum you can directly use it as index (int).

We can live with this enum34 backport package, and postpone this discussion.

**Comment by antnieszka** [comment]
@grzanka How about this: http://stackoverflow.com/a/36937 ?
Just simple class - readable and enum34 is no longer needed.

**Comment by grzanka** [comment]
At first glance looks OK.

**Comment by antnieszka** [comment]
Ok, I'll implement this and remove enum34.

**Comment by antnieszka** [comment]
@grzanka after looking at the code - do we need this class at all? It is not used anywhere. :open_mouth:

**Comment by grzanka** [comment]
which class ?

**Comment by antnieszka** [comment]
[Axis from curve.py](https://github.com/grzanka/beprof/blob/master/beprof/curve.py#L13)

**Comment by grzanka** [comment]
If its not used, simply remove it. I suspect that we will need it in the future, but then we will try to face this problem again.

**Comment by antnieszka** [comment]
Ok, removal commit incoming...
